packages/flutter/lib/src/gestures/drag_details.dart

```diff
@@ -52,9 +52,15 @@ class DragStartDetails {
   /// Creates details for a [GestureDragStartCallback].
   ///
   /// The [globalPosition] argument must not be null.
-  DragStartDetails({ this.globalPosition: Offset.zero })
+  DragStartDetails({ this.sourceTimeStamp, this.globalPosition: Offset.zero })
     : assert(globalPosition != null);
 
+  /// Recorded timestamp of the source pointer event that triggered the drag
+  /// event.
+  ///
+  /// Could be null if triggered from proxied events such as accessibility etc.
```

Comment:
"such as ... etc" is redundant, you can remove the "etc".

Reply:
Done

Reply:
you missed one. :-)

Reply:
oops, done